mandatory/1-syntax-errors.js

```diff
@@ -1,14 +1,14 @@
 // There are syntax errors in this code - can you fix it to pass the tests?
 
-function addNumbers(a b c) {
+function addNumbers(a, b, c) {
   return a + b + c;
 }
 
-function introduceMe(name, age)
-  return "Hello, my name is " + name "and I am " age + "years old";
-
+function introduceMe(name, age) {
+  return "Hello, my name is " + name + "and I am " + age + "years old";
+}
 function getTotal(a, b) {
```

Comment:
hi, please test this function in dev console

Reply:
Changed it, now it have spaces B)